mlflow/models/evaluation/default_evaluator.py

```diff
@@ -332,6 +347,48 @@ def __validate_artifacts(artifacts):
     )
 
 
+def _infer_artifact_type_and_ext(artifact_name, raw_artifact, exception_header):
+    # Given a path-like object: a string representation of path, or a pathlib.Path object
+    if isinstance(raw_artifact, str):
+        raw_artifact = pathlib.Path(raw_artifact)
+    if isinstance(raw_artifact, pathlib.Path):
+        ext_to_artifact_map = {
```

Comment:
should these enumeration dictionaries be stored externally to these functions (perhaps as constants in `mlflow.models.evaluation.artifacts` so that they could be easily referenced elsewhere?